Automattic/sensei
# Simplify the Pre-Merge Checklist

## Proposed Changes
As discussed in a Nexus team meeting, we are simplifying the Pre-Merge Checklist and removing some items that are best handled by the CI or can't be easily measured.

## Testing Instructions
[View the changed template](https://github.com/Automattic/sensei/blob/fd30224ec25cf1251da8920b5c9c5062833c4f9d/.github/PULL_REQUEST_TEMPLATE.md) and ensure the following items have been removed:
 - Data is sanitized / escaped
 - Test cases are written and passing (including feature flags)
 - We are generally happy with it and don’t feel like it immediately needs to be rewritten
 - Application performance has not degraded
 - Continuous Integration build is passing

## Pre-Merge Checklist
